src/formatters/deleteResultFormatter.ts

```diff
@@ -74,7 +74,8 @@ export class DeleteResultFormatter implements Formatter<DeleteSourceJson> {
           fullName: { header: 'FULL NAME' },
           type: { header: 'TYPE' },
           filePath: { header: 'PROJECT PATH' },
-        }
+        },
```

Comment:
I think I'd rather have the commands instantiate a new sf-plugins/core Ux and pass that to the formatters' constructor.  Then future style/ui changes become one-place-to-change.

There's some other UI inconsistencies between Ux and the SfCommand (colored warnings, for example) that we'd get consistency on.

Reply:
I looked into that, but it kept expanding what was required when implementing it for the now deprecated `project deploy` command. 

This approach will be the smallest change in the plugin, and if we wanted a one-place-to-change it, we would change this at the `oclif` level 🤷‍♂️

Reply:
Or maybe if there was another level of inheritance involved that provided common options for all source commands and formatters it could be changed in that one place.  Maybe call it, `SourceCommand` or `ResultFormatter`.